lib/openscap/xccdf/item.rb

```diff
@@ -64,6 +64,17 @@ def rationale(prefered_lang = nil)
         rationale
       end
 
+      def idents
+        idenss = []
+        idenss_it = OpenSCAP.xccdf_rule_get_idents(@raw)        
```

Comment:
I noticed trailing whitespace here.

It's nitpick, but I would feel better if we could get these fixed.